shared/Resource.go

```diff
@@ -57,7 +72,7 @@ const (
 	ScaleNone = "none"
 )
 
-func (r Resources) Validate() error {
+func (r Resources) Validate() error {//TODO replicas hinzufügen?
```

Comment:
Yes the replicas should be validated in this as well, otherwise the ReplicaResources.Validate() function doesn't get called anywhere

Reply:
Pushed the changes